src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -64,7 +64,7 @@ public void testPeekaboo() {
 	@Test
 	public void testCountOut() {
 		defaultCountingOutRhymer rhymer = new defaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = 0;
+		final int EMPTY_STACK_VALUE = -1;
```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).